src/Sentry.Unity.Editor/ConfigurationWindow/OptionsConfigurationTab.cs

```diff
@@ -70,21 +70,29 @@ internal static void CreateScript()
 
             var scriptName = Path.GetFileNameWithoutExtension(scriptPath);
 
-            File.WriteAllText(scriptPath, $@"using Sentry.Unity;
+            File.WriteAllText(scriptPath, $@"using System;
+using Sentry.Unity;
 using UnityEngine;
 
+/// Learn more at https://docs.sentry.io/platforms/unity/configuration/options/#programmatic-configuration
 [CreateAssetMenu(fileName = ""Assets/Resources/Sentry/{scriptName}.cs"", menuName = ""Sentry/{scriptName}"", order = 999)]
 public class {scriptName} : ScriptableOptionsConfiguration
 {{
-    // This method gets called when you instantiated the scriptable object and added it to the configuration window
-    public override void Configure(SentryUnityOptions options)
+    /// See base class for documentation.
+    public override void ConfigureAtBuild(SentryUnityOptions options)
     {{
-        // NOTE: Native support is already initialized by the time this method runs, so Unity bugs are captured.
-        // That means changes done to the 'options' here will only affect events from C# scripts.
+        // TODO implement
+        throw new NotImplementedException();
```

Comment:
```suggestion
```
Personally, I'd like to avoid causing errors due to incomplete SDK setup and just leave the comment there. IIRC this would break builds simply by having added this scriptable object to the configuration window.